hrms/svc/service.py

```diff
@@ -0,0 +1,60 @@
+from hrms.prst import model
+from sqlmodel import Session, select
+
+class Employee:
+    def list(self):
+        with Session(model.engine) as session:
+            statement = select(model.Employee)
+            results = session.exec(statement)
+            employees = results.all()
+            return employees
+        
+    def get(self, id: int):
+        with Session(model.engine) as session:
+            statement = select(model.Employee).where(model.Employee.id == id)
+            results = session.exec(statement)
+            record = results.first()
+            if record is not None:
+                return record
+            else:
+                return {"message": "record not found"}
+
+    def add(self, name: str, designation: str, reports_to: int, project: str):
+        employee = model.Employee(name=name, designation=designation, reports_to=reports_to, project=project)
+        with Session(model.engine) as session:
+            session.add(employee)
+            session.commit()
+            return {"message": "employee added successfully"}
+
+    def edit(self, id: int, name: str = None, designation: str = None, reports_to: int = None, project: str = None):
```

Comment:
In an ideal case the application/business-logic treats whole of the employee as a single unit so it is better to use the whole object instead of individual attributes.